hassio/src/snapshots/hassio-snapshots.ts

```diff
@@ -83,8 +83,8 @@ class HassioSnapshots extends LitElement {
           Create snapshot
         </h1>
         <p class="description">
-          Snapshots allow you to easily backup and restore all data of your
-          Hass.io instance.
+          Snapshots allow you to easily backup and restore all data of your Home
```

Comment:
Should have access to `hass` here, prefer localize

Reply:
True for all files touched

Reply:
Yes, but that's out of scope for this PR, which is just about applying the rebrand.